projects/plugins/jetpack/modules/sso/class.jetpack-sso-user-admin.php

```diff
@@ -445,6 +475,16 @@ public function render_custom_email_message_form_field( $type ) {
 			}
 		}
 
+		/**
+		 * Conditionally disable the core invitation email.
+		 * It should be sent when SSO is disabled or when admins opt-out of WordPress.com invites intentionally.
+		 *
+		 * @return boolean Indicating if the core invitation main should be sent.
+		 */
+		public function send_wp_mail_new_user() {
```

Comment:
Can we change the name of this function? Something like `should_send...`, because as it is I expect this function to perform the sending!